__tests__/Unit/Components/Tasks/SuggestionBox.test.tsx

```diff
@@ -0,0 +1,50 @@
+import SuggestionBox from '@/components/tasks/SuggestionBox/SuggestionBox';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { DUMMY_PROFILE as placeholderImageURL } from '@/constants/display-sections';
+import { GithubInfo } from '@/interfaces/suggestionBox.type';
+
+const gitInfo: GithubInfo[] = [
+    {
+        github_id: 'fakhruddinkw',
+        profileImageUrl: placeholderImageURL,
+    },
+    {
+        github_id: 'iamitprakash',
+        profileImageUrl: placeholderImageURL,
+    },
```

Comment:
Nit: best to use names like John doe/ Jane doe instead of using specific RDS members.